### ISSUE TITLE
CaptureEntirePageScreenshotAndWait And CaptureEntirePageScreenshot is not working

### ISSUE THREAD

**Issue by GoogleCodeExporter** [prajaktamoghe/selenium-vba#28]
```
Operating system and version (32/64bit) :
.Net Framework version :
Office name and version(32/64bit) :
Browser name and version :
SeleniumWrapper version :

What steps will reproduce the problem with a public website ?
1. go to any website using firefox ide
2. use captureEntirePageScreenshot or captureEntirePageScreenshotAndWait and 
set the target in a folder (ex: 
C:\\Users\\me\\Documents\\Screenshot\\image1.jpg)
3. run the vba code that you get from the source tab in selenium ide into vba 
Excel

What is the expected output? What do you see instead?

the picture should be saved in the path

Please provide any additional information below.


once the code is generated we get an error message in vba excel saying: 


```

Original issue reported on code.google.com by `alam...@gmail.com` on 18 Sep 2013 at 8:15

Attachments:
- [seleniumErrorMessage.jpg](https://storage.googleapis.com/google-code-attachments/selenium-vba/issue-28/comment-0/seleniumErrorMessage.jpg)

**Comment by GoogleCodeExporter** [comment]
```
The "CaptureEntirePageScreenshot" command is not implemented, but will be in 
the next release.

You can use this one instead which does the same thing:
driver.getScreenshot().SaveAs "c:\capture.png"
```

Original comment by `florentbr` on 18 Sep 2013 at 2:24
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in version 1.0.15.0
```

Original comment by `florentbr` on 18 Oct 2013 at 3:43
- Changed state: **Fixed**
